### DISCUSSION TITLE
A button for mask editor?

### DISCUSSION THREAD

**Discussion by RoyDingZF** [comfyanonymous/ComfyUI#3688] [Ideas]
I think the mask editor is a frequently used function, why not add a button on load image node to open the mask editor for convenience? right clicking on the node and look it up every time is cumbersome.

**Comment by ktee95** [comment]
same suggestion

**Comment by christian-byrne** [comment]
This was just added by a contributor in https://github.com/Comfy-Org/ComfyUI_frontend/pull/3603. It is on the latest version of the frontend.
